pkg/buildpod/generator.go

```diff
@@ -145,12 +145,18 @@ func (g *Generator) fetchBuilderConfig(build BuildPodable) (v1alpha1.BuildPodBui
 		return v1alpha1.BuildPodBuilderConfig{}, err
 	}
 
+	config, err := image.ConfigFile()
```

Comment:
maybe don't need this with the os in the builder status now

Reply:
we don't have the builder resource at this point, just the builder image so I don't think I can get that status unless I fetch the resource